src/visualizer/controller.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+    const {
+        getListOfData,
+        getData
+    } = window.comm;
+    const {
+        createGroupInfoElement,
+        createAverageValueTable
+    } = window.detailsElement;
+    const {
+        createGroupSvg
+    } = window.svg;
+
+    getListOfData().then(responds => {
+        console.log(responds);
+        createButtons(responds);
+    });
+
+    /**
+     * @summary Creates a button for each file in the data folder
+     * @param {Array} array this is an array with the names of the files in the data folder
+     */
+    function createButtons(array) {
+        for (let i = 0; i < array.length; i++) {
+            if (array[i] !== ".gitkeep"){
+                const buttonElement = document.createElement("BUTTON");
+                buttonElement.innerText = "Filename: " + array[i];
+                buttonElement.addEventListener("click", () => callData(array[i]));
+                document.body.appendChild(buttonElement);
```

Comment:
Jeg er ikke så glad for at i appender shit til dom'en.
Kan i ikke fyre alle de her knapper ind i et div og return det.
Så kan i appende det til dom'en i den funktion der kalder den her

Reply:
Done